.travis.yml

```diff
@@ -30,7 +30,9 @@ matrix:
           install: pip3 install -r tests/client-dbus/requirements.txt
           before_script:
               - cd tests/client-dbus
-          env: TASK=lint
+          script:
```

Comment:
Please just keep separate, one task per matrix entry. It makes things simpler.

Reply:
You are suggesting this?
 
       - language: python
          python: "3.4"
          install: pip3 install -r tests/client-dbus/requirements.txt
          before_script:
              - cd tests/client-dbus
          env: TASK=lint
        - language: python
          python: "3.4"
          install: pip3 install -r tests/client-dbus/requirements.txt
          before_script:
              - cd tests/client-dbus
          env: TASK=fmt-travis

Reply:
If that's possible...tasks not dependent on each other.